docs/rules/accessibility/no-title-attribute-counter.md

```diff
@@ -10,7 +10,24 @@ If you are considering the`title` attribute to provide supplementary description
 
 ### Should I use the `title` attribute to provide an accessible name for an `<svg>`?
 
-Use a `<title>` element instead of the `title` attribute, or an `aria-label`.
+To label a _simple_ SVG that is purely graphical, we recommend the following technique for greatest assistive technology support:
+
+- Set `role="img"` on the SVG
+- Set the first child of the SVG to a `title` element that provides the accessible name for the SVG.
+- Re-inforce that the `title` is the accessible name for the SVG by associating it with an `aria-labelledby` on the SVG.
+
+Example:
+
+```.html
+<svg role="img" aria-labelledby="titleText">
+  <title id="titleText">Online</title>
+  ...
+</svg>
+```
+
+Alternatively, if you do not care to have the native browser tooltip that is only accessible by mouse, you can use `aria-label` to label the SVG instead of `title`.
```

Comment:
something that was called out [in slack](https://github.slack.com/archives/C0FSWLQ0Y/p1659571663703429?thread_ts=1659548483.769949&cid=C0FSWLQ0Y)

>  I do think that if <title> is used, and provides hover behaviour but can't be revealed by keyboard users, that is a WCAG failure though.

Reply:
Yes, I think we can't suggest this pattern because of that violation (sad that I didn't notice it as an issue originally!)

Let's brainstorm some other ideas in Slack and come up with a definitive solution to this problem!